src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,100 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0 150px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 38px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column: 2 / span 11;
```

Comment:
Could you make this span 10, not 11 columns? Although this won't affect the layout of the current content today, if we changed the some of the other stuff, it could cause the cards to start taking up space right up until the right edge of the black container. You can see the asymmetry in the following screenshot:

<img width="1341" alt="Screen Shot 2020-11-11 at 10 44 50 PM" src="https://user-images.githubusercontent.com/1002748/98905115-b7e2dc80-246f-11eb-80e1-518f535d75ca.png">


```suggestion
  grid-column: 2 / span 10;
```

Reply:
👍 Changed back to span 10. However, the width of the statements is affected and does not end/start at the correct word compared to what is shown in Figma.